ssz/cache/cache.py

```diff
@@ -0,0 +1,51 @@
+from collections.abc import (
+    MutableMapping,
+)
+from typing import (
+    TYPE_CHECKING,
+    Iterator,
+)
+
+from lru import LRU
+
+if TYPE_CHECKING:
+    MM = MutableMapping[bytes, bytes]
+else:
+    MM = MutableMapping
+
+
+class SSZCache(MM):
```

Comment:
I see, now `SSZCache` only serves as an example of how can the upper level (Trinity) implement a new `MutableMapping` subclass instead of simple `LRU()`. e.g., if we want to add TTL.